CHANGELOG.md

```diff
@@ -76,6 +76,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#1042](https://github.com/nix-rust/nix/pull/1042))
 - Add `sched_yield`
   ([#1050](https://github.com/nix-rust/nix/pull/1050))
+- Add `unlinkat`
```

Comment:
Your changelog entry got dislocated during a rebase.

Reply:
Hi ... what do you mean by dislocated? For the latest patch this is the change I intended for changelog.md:

-  ([#TBD](https://github.com/nix-rust/nix/pull/1058))
+  ([#1058](https://github.com/nix-rust/nix/pull/1058))

Is this what you can see? Is there another push needed for this file?